src/test/java/com/wikia/webdriver/testcases/mercurytests/recentwikiactivity/RecentWikiActivityTest.java

```diff
@@ -0,0 +1,95 @@
+package com.wikia.webdriver.testcases.mercurytests.recentwikiactivity;
+
+import com.wikia.webdriver.common.contentpatterns.URLsContent;
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.core.annotations.Execute;
+import com.wikia.webdriver.common.core.annotations.InBrowser;
+import com.wikia.webdriver.common.core.api.ArticleContent;
+import com.wikia.webdriver.common.core.drivers.Browser;
+import com.wikia.webdriver.common.core.helpers.Emulator;
+import com.wikia.webdriver.common.core.helpers.User;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.elements.mercury.pages.RecentWikiActivityPage;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.testng.annotations.Test;
+
+@Execute(onWikia = "aga")
+@InBrowser(browser = Browser.CHROME, emulator = Emulator.GOOGLE_NEXUS_5)
+public class RecentWikiActivityTest extends NewTestTemplate {
+
+  @FindBy(css = ".diff-page__undo")
+  private WebElement undoButton;
+
+  @Test()
+  public void recentWikiActivity_openFromHamburgerMenu() {
```

Comment:
plaease provide test setup in the name, make it more business value oriented, and remove the "recentWikiActivity"
so that it would be AnonUser_canOpenRecentWikiActivityFromHamburgerMenu
Please add a test group. Or you don't want to do so for some reason ?